charts/osm/templates/osm-bootstrap-deployment.yaml

```diff
@@ -57,10 +57,14 @@ spec:
         - name: init-osm-bootstrap
           image: "{{ include "osmCRDs.image" . }}"
           imagePullPolicy: {{ .Values.osm.image.pullPolicy }}
-          args:
-          - apply
-          - -f
-          - /osm-crds
+          command:
+          - sh
+          - -c
+          - >
+            kubectl apply -f /osm-crds;
+            {{- if .Values.osm.enableReconciler }}
+            kubectl label -f /osm-crds openservicemesh.io/reconcile=true;
+            {{- end }}
```

Comment:
If this is the plan moving forward we should probably change the [osm-crds Dockerfile](https://github.com/keithmattix/osm/blob/6b3da7b8b82da6c081e991904d847de2d8982960/dockerfiles/Dockerfile.osm-crds#L10) to no longer have `kubectl` as the entrypoint.

Reply:
Defining a `command` for a container in a pod overrides `ENTRYPOINT` in the Dockerfile, so this still works the way we want. I can see marginal value in removing it from the Dockerfile for the sake of general cleanliness, but I don't see it causing any issues as long as that image stays an implementation detail and is mostly unexposed to users.

Reply:
Yeah, agreed. I should have specified, it wasn't from a functional standpoint but likely we should have the Dockerfile reflecting how the image should be used.

Reply:
I found out the osm-crds image is apparently also used for some of the e2e tests in GitHub actions, so I'm wondering if changing the `ENTRYPOINT` will break that workflow.

Reply:
Based on where I see us using it in the chart I don't think changing it will break anything since both places define a `command`:
https://github.com/openservicemesh/osm/blob/8bae12f7186d3bf7709951ba8c37e68e569f30c8/charts/osm/templates/cleanup-hook.yaml#L77
https://github.com/openservicemesh/osm/blob/8bae12f7186d3bf7709951ba8c37e68e569f30c8/charts/osm/templates/osm-bootstrap-deployment.yaml#L58

I can't think of any other place where that image is run.